pkg/kubectl/cmd/cp.go

```diff
@@ -299,6 +299,16 @@ func (o *CopyOptions) copyFromPod(src, dest fileSpec) error {
 // stripPathShortcuts removes any leading or trailing "../" from a given path
 func stripPathShortcuts(p string) string {
 	newPath := path.Clean(p)
+	trimmed := strings.TrimPrefix(newPath, "../")
+
+	for trimmed != newPath {
+		newPath = trimmed
+		trimmed = strings.TrimPrefix(newPath, "../")
+	}
+
+	// trim leftover ".."
+	newPath = strings.TrimPrefix(newPath, "..")
```

Comment:
won't this incorrectly remove ".." from "...foo"?

Reply:
add a testcase for a path containing three leading dots, since that has cropped up a few times as a mishandled case